app/src/main/java/openfoodfacts/github/scrachx/openfood/views/MainActivity.java

```diff
@@ -541,4 +566,63 @@ public void moveToBarcodeEntry(){
             Log.e("MainActivity", "Error in creating fragment");
         }
     }
+    /*  Method to register runtime broadcast receiver to show snackbar alert for internet connection..
+          */
+    private void registerInternetCheckReceiver() {
+        IntentFilter internetFilter = new IntentFilter();
+        internetFilter.addAction("android.net.wifi.STATE_CHANGE");
+        internetFilter.addAction("android.net.conn.CONNECTIVITY_CHANGE");
+        registerReceiver(broadcastReceiver, internetFilter);
+    }
+
+    /**
+     *  Runtime Broadcast receiver inner class to capture internet connectivity events
+     */
+    public BroadcastReceiver broadcastReceiver = new BroadcastReceiver() {
+        @Override
+        public void onReceive(Context context, Intent intent) {
+            String status = utils.getConnectivityStatusString(context);
+            setSnackbarMessage(status,false);
+        }
+    };
+
+    private void setSnackbarMessage(String status,boolean showBar) {
+        String internetStatus="";
+        if(status.equalsIgnoreCase("Wifi enabled")||status.equalsIgnoreCase("Mobile data enabled")){
+            internetStatus="Network Status: Connected";
+        }else {
+            internetStatus="Network Status: Disconnected";
```

Comment:
Can you make these strings resources? This will help with translation.

Reply:
Can you help me ?

Reply:
Refer http://envyandroid.com/android-studio-extract-strings-resources/